live-flowchart/README.md

```diff
@@ -9,19 +9,19 @@ Advanced
 
 Register + Unregister
 
-![Screenshot](https://raw.githubusercontent.com/franciov/serviceworker-cookbook/recipe/live-flowchart/live-flowchart/register-unregister.png)
+![Screenshot](https://raw.githubusercontent.com/mozilla/serviceworker-cookbook/recipe/live-flowchart/live-flowchart/register-unregister.png)
```

Comment:
https://raw.githubusercontent.com/mozilla/serviceworker-cookbook/recipe/live-flowchart/live-flowchart/register-unregister.png
should be:
https://raw.githubusercontent.com/mozilla/serviceworker-cookbook/master/live-flowchart/register-unregister.png